02_programming_fundamentals/04_algorithms_day_2/02_range_function/01_range.js

```diff
@@ -0,0 +1,11 @@
+// Write a function that take two parameters and return a new array with defaults values beetwenn the start
```

Comment:
// Write a function that takes two input parameters, and returns a new array with defaults values between the start
 // number and the end number.
// This function should work in both ascending or descending order.